test cases/rust/20 transitive dependencies/foo.c

```diff
@@ -0,0 +1,6 @@
+int foo_rs(void);
```

Comment:
Not that it actually matters but the signedness is different between your C prototype and the Rust. Maybe use `unsigned` instead? Or make the rust `i32`?

Reply:
indeed, fixed.